editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -111,19 +111,19 @@ import {
   isExported,
   isDefaultExport,
   expressionTypeForExpression,
+  extractPrefixedCode,
 } from './parser-printer-parsing'
 import { getBoundsOfNodes, guaranteeUniqueUidsFromTopLevel } from './parser-printer-utils'
 import { ParserPrinterResultMessage } from './parser-printer-worker'
-import creator from './ts-creator'
```

Comment:
does this mean Sean doesn't have to sacrifice a goat once every week anymore?

Reply:
It indeed means exactly that! This dependency will go in the bin. I didn't remove it yet for fear of merge conflicts with Sean's ongoing work, but that has been merged to master now so I will remove it as part of this PR